### PR TITLE
feat: quotas

### LINKED ISSUE #320: feat: quotas
Integrate the new version of `cerberus` which has `ProjectData::quota` attribute. And limit access to projects that exceeded the quota.

### PR DESCRIPTION
**Pull request by xDarksome**
# Description

Resolves #320 

- bumps `cerberus` to 0.10.0
- adds Grafana dashboard for the amount of quota limited requests
- fixes ProjectData cache deserialization errors resulting in 500
- fixes missing secret in `ci_terraform.yaml`

## How Has This Been Tested?

Existing integration tests (staging)

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: src/state.rs [hunk 1]
```diff
@@ -45,4 +48,29 @@ impl AppState {
...
+            .project_data(id)
+            .await
+            .tap_err(|_| self.metrics.add_rejected_project())?;
+
+        project.validate_access(id, None).tap_err(|e| {
+            self.metrics.add_rejected_project();
+            info!("Denied access for project: {id}, with reason: {e}");
+        })?;
+
+        if !project.quota.is_valid {
```
Comment (chris13524) [inline src/state.rs:64]: Does this "quota" apply to all endpoints? History? Identity? I assume the quota is only incremented in cloud/data lake for RPC requests.

**Review by chris13524 — APPROVED** [review]
LGTM

File: Cargo.toml [hunk 2]
```diff
@@ -52,7 +52,7 @@ pnet_datalink = "0.31"
 tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["json", "ansi"] }
 
-cerberus = { git = "https://github.com/WalletConnect/cerberus.git", tag = "v0.9.1" }
```
Comment (geekbrother) [inline Cargo.toml:55]: Can you please add to the description that we are bumping a version of the cerberus in this PR?

**Review by geekbrother — APPROVED** [review]
Looks good for me!

File: src/state.rs (hunk 1 above)
Comment (xDarksome) [reply src/state.rs:64]: @arein mentioned that identity shouldn't be affected, not sure about history. 
@arein ?

Comment (xDarksome) [reply src/state.rs:64]: > Derek: No
> not for now. It may need a different quota later
So, I'm removing it for history

**Pull request merged**